packages/core/src/error/MessageSendingError.ts

```diff
@@ -1,10 +1,13 @@
-import type { OutboundMessage } from '../types'
+import type { OutboundDidCommV1Message, OutboundDidCommV2Message } from '../didcomm'
```

Comment:
Now in main branch we have a more generic 'OutboundMessageContext', which  I think that can be used for both V1 and V2 messages (some helper methods like isDidcommV1Message / isDidcommV2Message might be moved to that class as well).